### ISSUE TITLE
Crash at sync_fuzzers() when running without -E option (No crash with -E, but it gives another error)

### ISSUE THREAD

**Issue by mirzamomen** [aflnet/aflnet#65]
I detect a crash trying synchronisation (-S ) with a parallel aflnet fuzzer, when -E is NOT provided.
I am using commit 0f51f9e

and I run fuzzer as:
 /home/ubuntu/aflnet/afl-fuzz -S 1 -i /home/ubuntu/experiments/in-dns/ -o /home/ubuntu/out -N udp://127.0.0.1/5353 -P DNS -D 10000 -K ./dnsmasq

The backtrace is:
Core was generated by `/home/ubuntu/aflnet/afl-fuzz -S 1 -i /home/ubuntu/experiments/in-dns/ -o /home/'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x0000000000416c7d in send_over_network () at afl-fuzz.c:1069
1069        n = net_send(sockfd, timeout, kl_val(it)->mdata, kl_val(it)->msize);
(gdb) bt
#0  0x0000000000416c7d in send_over_network () at afl-fuzz.c:1069
#1  0x0000000000417b47 in run_target (argv=argv@entry=0x7ffc36bc4c28, timeout=40) at afl-fuzz.c:3263
#2  0x0000000000403541 in sync_fuzzers (argv=0x7ffc36bc4c28) at afl-fuzz.c:7732
#3  0x0000000000405e29 in main (argc=<optimized out>, argv=<optimized out>) at afl-fuzz.c:9271

However, if I provide -E option, like:

 /home/ubuntu/aflnet/afl-fuzz -S 1 -i /home/ubuntu/experiments/in-dns/ -o /home/ubuntu/out -N udp://127.0.0.1/5353 -P DNS -D 10000 -E -K ./dnsmasq

 I get this error:

SYSTEM ERROR : No server states have been detected. Server responses are likely empty!
    Stop location : main(), afl-fuzz.c:9167
       OS message : No such process

**Comment by thuanpv** [comment]
Hi @mirzamomen. Thanks for reporting the error. Did you set different ports for the fuzzing instances?

**Comment by mirzamomen** [comment]
Hi @thuanpv. Thanks for the reply. 
Yes, I did. The other fuzzer works with port 5354 like:

/home/ubuntu/aflnet/afl-fuzz -S 1 -i /home/ubuntu/experiments/in-dns/ -o /home/ubuntu/out -N udp://127.0.0.1/5354 -P DNS -D 10000 -K ./dnsmasq

But I just noticed that I should have run the "Dnsmasq" with --conf-file set to the new configuration file, in which port is changed to 5354, which I have missed. It ran successfully when I did so. 

Thanks again.
